### PR TITLE
Chaincode operations addition through fabric sdk

### PR DESCRIPTION
**Pull request by jopuneet**
- Chaincode approve and commit operations added
- REST API added for performing chaincode operations

Signed-off-by: Puneet Joshi <puneet.joshi@walmart.com>

### CODE REVIEW

File: src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    validateChaincodeOperationsInput(chaincodeOperationsModel, operationsType);
+    HFClient hfClient = HFClient.createNewInstance();
+    Network network = gateway.getNetwork(networkName);
+    Channel channel = network.getChannel();
+
+    switch (operationsType) {
+      case approve:
+        {
+          return approveChaincode(hfClient, channel, chaincodeOperationsModel);
```
Comment (arsulegai) [inline src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java:50]: Looks like a default instance of hfClient is created in this method and not initialized to any network or signer. It is then passed to the approve/commit.

**Review by arsulegai — APPROVED** [review]

File: src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      String networkName,
+      ChaincodeOperations chaincodeOperationsModel,
+      ChaincodeOperationsType operationsType) {
+
+    validateChaincodeOperationsInput(chaincodeOperationsModel, operationsType);
+    HFClient hfClient = HFClient.createNewInstance();
+    try {
+      hfClient.setCryptoSuite(CryptoSuite.Factory.getCryptoSuite());
+      X509IdentityProvider.INSTANCE.setUserContext(
+          hfClient, gateway.getIdentity(), "hlf-rest-client");
```
Comment (anandbanik) [inline src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java:51]: @jopuneet Can u change it to hlf-connector?

Comment (jopuneet) [reply src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java]: @anandbanik Done. thank you

File: src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java (hunk 1 above)
Comment (jopuneet) [reply src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java]: @arsulegai Done. thank you.

**Review by arsulegai — APPROVED** [review]

File: src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java (hunk 2 above)
Comment (anandbanik) [reply src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java]: @jopuneet Thanks!

**Review by anandbanik — COMMENTED** [review]
approved

**Pull request merged**